src/react_native/core.cljs

```diff
@@ -187,14 +187,18 @@
   (react/useMemo handler (get-js-deps deps)))
 
 (defn delay-render
-  [content]
+  [content & {:keys [ms] :or {ms 0}}]
   (let [[render? set-render] (use-state false)]
     (use-mount
      (fn []
-       (js/setTimeout #(set-render true) 0)))
+       (js/setTimeout #(set-render true) ms)))
     (when render?
       content)))
 
+(defn render-after
+  [{:keys [_ms] :as opts} & children]
+  (delay-render (into [:<>] children) opts))
+
```

Comment:
This is expected to be used as a component (`delay-render` is used as a function), also it receives any amount of children, compared to `delay-render` that receives a single child.

Sometimes we want to delay a bit more the components, `setTimeout` with `0` [is extremely fast](https://reactnative.dev/docs/0.73/timers#timers), since our current React version on Android don't batch updates, an option would be use `reagent/next-tick` or `js/RequestAnimationFrame` instead.

Since some components need a bigger delay, this `render-after` component receives an `ms` key and renders after that period of time. (ms used to match the `dispatch-later` key from re-frame, so I don't introduce a new concept).